src/services/embedding/adapters/base.py

```
@@ -61,24 +61,38 @@ class BaseEmbeddingAdapter(ABC):
+    def _validate_texts(self, texts: list[str]) -> None:
+        """
+        Validate that texts list is not empty.
+
+        Args:
+            texts: List of texts to validate
+
+        Raises:
+            ValueError: If texts list is empty
+        """
+        if not texts:
+            raise ValueError("Embedding request requires at least one text")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)